### PR TITLE
Fix full view toggle icon styles

### PR DESCRIPTION
**Pull request by VkoHov**
- Change icon styles. it is visible despite background color.

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1475](https://github.com/aimhubio/aim/issues/1475)

**Review by rubenaprikyan — APPROVED** [review]
lgtm

File: aim/web/ui/src/styles/abstracts/_variables.scss [hunk 1]
```diff
@@ -90,6 +90,7 @@ $error-color-5: #f7f1f5;
 $border-color-main: $primary-color-10;
 $border-main: 0.0625rem solid $border-color-main;
 $border-separator: 0.0625rem solid $primary-color-20;
+$border-main-darker: 0.0625rem solid $primary-color-40;
```
Comment (KaroMourad) [inline aim/web/ui/src/styles/abstracts/_variables.scss:93]: can't see this variable `border-main-darker` in usage, is it necessary?
if not let's remove it

Comment (VkoHov) [reply aim/web/ui/src/styles/abstracts/_variables.scss:93]: There are a lot of variables without usage, it's our set of SCSS variables and this one is one of them that we forgot to add. I don't think that we have to remove it. Unused SCSS variables wouldn't be included in built CSS files.

**Review by KaroMourad — APPROVED** [review]
lgtm

**Pull request merged**